Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_AuthorShow.1.txt

```diff
@@ -83,100 +83,140 @@
             <a href="https://example.com/owner/name.git">
               <h4>vapor-1</h4>
               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas nec orci scelerisque, interdum purus a, tempus turpis.</p>
-              <small>
-                <span class="identifier">owner/name</span> &ndash; 
-                <span class="stars">3</span>
-              </small>
+              <ul class="metadata">
+                <li class="identifier">
+                  <small>owner/name</small>
+                </li>
+                <li class="stars">
+                  <small>3 stars</small>
+                </li>
```

Comment:
This diff and the corresponding image don't show any last activity info - I'm guessing that's because it's `nil` in the mock data, right? Just confirming, I don't think we need to add that necessarily.

Reply:
Actually, I think I will add a date here as a secondary validation that the `packageList` work I did is being used in all these places.